nav2_map_server/src/costmap_filter_info/costmap_filter_info_server.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2020 Samsung Research Russia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO(AlexeyMerzlyakov): This dummy info publisher should be removed
+// after Semantic Map Server having the same functionality will be developed.
+
+#include "nav2_map_server/costmap_filter_info_server.hpp"
+
+#include <string>
+#include <memory>
+#include <utility>
+
+namespace nav2_map_server
+{
+
+CostmapFilterInfoServer::CostmapFilterInfoServer()
+: nav2_util::LifecycleNode("costmap_filter_info_server")
+{
+  declare_parameter("filter_info_topic", "costmap_filter_info");
+  declare_parameter("type", 0);
+  declare_parameter("mask_topic", "map_mask");
```

Comment:
I think a better default is `filter_mask` since this isn't really a "map"

Reply:
This was named "map_mask" as meaning of "mask applied on map" or just "map's mask". Also, "map_mask" name hints that this topic has the same as map type: OccupancyGrid. Just confirm me to change it: there is no problem to make it. But in my opinion, "map_mask" is also a good name.

Do I need to update the terminology to "filter mask" instead of "map mask" everywhere, including documentation as well?

Reply:
I could live with it as is, I leave it to you. But I think terminology would be more reasonable as filter_mask since this is a `mask` being used by a given `costmap filter`, and while it is in most cases a `map`, the "mask" word seems to have the contextual meaning as "map".

Reply:
The terminology was updated from "map mask" -> to "filter mask".